pygmt/figure.py

```diff
@@ -289,6 +289,9 @@ def savefig(
         dpi : int
             Set raster resolution in dpi. Default is 720 for PDF, 300 for
             others.
+        **kwargs : dict
+            Additional keyword arguments passed to
+            :meth:`pygmt.Figure.psconvert`.
```

Comment:
I find it a bit confusing that `dpi` was previously an extra option or kwarg, but not there's both `dpi` and `**kwargs` in the `Figure.savefig(...)` method shown in the documentation:
![image](https://user-images.githubusercontent.com/23487320/186732775-106c274c-2ff7-4734-833d-c94fcd7b4040.png).

Not sure what's the best solution here. Removing the `dpi` parameter documentation doesn't seem ideal, nor does including all of `psconvert`'s parameters in `savefig`. Would it make sense then to put the `dpi` documentation under `**kwargs`? Or is there a cleaner solution?

Reply:
I agree. Actually some of `psconvert`'s parameters (e.g., `prefix`, `fmt`) can **NOT** be used in `Figure.show` or `Figure.savefig`.

Perhaps we should say:
```
**kwargs: dict
    Additional keyword arguments passed to `pygmt.Figure.pscovert`. 
    Valid parameters are `gs_path`, `gs_option`, `resize`, `bb_style`, and `verbose`.
```

Reply:
Yes! Also need to do a similar thing for the fig.show docstring.